testsuite/python/tests_common.py

```diff
@@ -81,6 +81,7 @@ def func(self):
         # set Parameter
         Inter = interClass(**params)
         Inter.validate_params()
+        Inter._set_params_in_es_core()
```

Comment:
On Wed, Sep 07, 2016 at 11:42:15AM -0700, Michael Kuron wrote:
> > @@ -81,6 +81,7 @@ def func(self):
> >          # set Parameter
> >          Inter = interClass(**params)
> >          Inter.validate_params()
> > +        Inter._set_params_in_es_core()
> 
> Don't do that. Methods prefixed with `_` are private and should not be called from outside the class.
Actually, by now, it should be possible to actiave and dactiave
actors by adding and remong them from system.actors.
See the coulomb_cloud_wall.py testcase.
At the point in time when I wrote the decorators for the actor
tests, that was not possible, so original test did not cover actual
activation, just interface consistency.